lib/browser_sniffer/patterns.rb

```diff
@@ -62,6 +63,8 @@ class BrowserSniffer
       ], [:name, :version, :major, [:type, :ie]], [
         /Mozilla\/5.0.*Windows NT 6\.\d.*Trident\/7\.\d.*rv:(\d+)\.\d*/i #IE11 on Win7
       ], [:major, [:version, 7], [:name, 'Internet Explorer'], [:type, :ie]], [
+        /Mozilla\/5.0.*Windows NT 10\.\d.*Trident\/7\.\d.*rv:(\d+)\.\d*.*like\sGecko/i #IE11 on Win10
```

Comment:
Per the [MSDN article](https://msdn.microsoft.com/en-us/library/hh869301(v=vs.85).aspx#ie11):

>The "like Gecko" token has been added (for consistency with other browsers).

So, I'm just adding it to tighten up the regex match to this specific user agent string.